### ISSUE TITLE
Feature request: Include asset details and description in Asset object

### ISSUE THREAD

**Issue by UltimateForm** [stackbit/sourcebit-source-contentful#18]
Right now Asset objects are normalized to only have partial information, they do not contain the details and description which can be very useful to have

Please include these
Here's an example of how they look like in the API
```json
                "fields": {
                    "title": "Sparkler",
                    "description": "John with Sparkler",
                    "file": {
                        "url": "//images.ctfassets.net/spaceid/7orLdboQQowIUs22KAW4U/e237faf0c6a0c89f8dce3e35e552176e/matt-palmer-254999.jpg",
                        "details": {
                            "size": 2293094,
                            "image": {
                                "width": 3000,
                                "height": 2000
                            }
                        },
                        "fileName": "matt-palmer-254999.jpg",
                        "contentType": "image/jpeg"
                    }
                }
```

**Comment by UltimateForm** [comment]
I have opened a PR #19  to have this collect the details object, didn't include the description yet

**Comment by hsemedo** [comment]
@smnh any plan regarding release of changes included in #19 ?

**Comment by smnh** [comment]
Published new v0.6.8 version

**Comment by smnh** [comment]
Publish new version [v0.6.8](https://github.com/stackbit/sourcebit-source-contentful/releases/tag/v0.6.8)
